.vsts-pipelines/templates/steps/script.yml

```diff
@@ -0,0 +1,70 @@
+# File: templates/steps/script.yml
+
+parameters:
+  name: 'DO NOT LEAVE DEFAULT'
```

Comment:
maybe you could name this `artifactName` and enable the `autoArtifact` step if it's not empty?